### ISSUE TITLE
[Combobox][A11y][Expected Fixed Time: ASAP]: In JAWS with VPC mode, the VPC focus goes onto the decorative arrow-down icon inside the Combobox

### ISSUE THREAD

**Issue by lv-sap** [SAP/ui5-webcomponents#10761]
### Bug Description

With JAWS in VPC mode, the VPC focus goes onto the decorative "Select Options" arrow-down icon inside the Combobox.

![Image](https://github.com/user-attachments/assets/0a7998cf-177a-4ca3-8b14-cad20e2e5954)

### Affected Component

Combobox

### Expected Behaviour

Per A11y guidelines and request, the arrow-dwon icon here should be treated as a decorative image.

### Steps to Reproduce

1. Go to https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpOyBoZWlnaHQ6IDMwMHB4XCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1jb21ib2JveCBwbGFjZWhvbGRlcj1cIkVudGVyIHZhbHVlXCI-XG4gICAgICAgIDx1aTUtY2ItaXRlbSB0ZXh0PVwiQXVzdHJpYVwiPjwvdWk1LWNiLWl0ZW0-XG4gICAgICAgIDx1aTUtY2ItaXRlbSB0ZXh0PVwiQnVsZ2FyaWFcIj48L3VpNS1jYi1pdGVtPlxuICAgICAgICA8dWk1LWNiLWl0ZW0gdGV4dD1cIkdlcm1hbnlcIj48L3VpNS1jYi1pdGVtPlxuICAgICAgICA8dWk1LWNiLWl0ZW0gdGV4dD1cIkl0YWx5XCI-PC91aTUtY2ItaXRlbT5cbiAgICAgICAgPHVpNS1jYi1pdGVtIHRleHQ9XCJTcGFpblwiPjwvdWk1LWNiLWl0ZW0-XG4gICAgPC91aTUtY29tYm9ib3g-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9Db21ib0JveC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvQ29tYm9Cb3hJdGVtLmpzXCI7In19
2. Enable JAWS in VPC mode
3. Click the arrow down icon

### Priority

None

### UI5 Web Components Version

^2.4.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by NHristov-sap** [comment]
Hello @lv-sap ,

I've tried the reproducing steps ypu provided, and cannot reproduce the issue - the focus remains in Input part of the Combobox:

![Image](https://github.com/user-attachments/assets/a9a7ab69-1029-494d-a428-3c0fd39332f5)

Please check again.

Best Regards,
Nikolay Hristov

**Comment by lv-sap** [comment]
@NHristov-sap , As I mentioned in the description, you need to use JAWS (or maybe NVDA) screen reader in VPC mode to reproduce the issue.
Basically, what the a11y testing person expects is to "...ensure search icon announcement is hidden to screen reader". And for that, the decorative "caret" (arrow-down) icon should be treated as a "decorative image".

**Comment by NHristov-sap** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

Please check the reported issue.

Best Regards,
Nikolay Hristov
